backend/plugin/db/mysql/sync.go

```diff
@@ -470,6 +477,147 @@ func isCurrentTimestampLike(s string) bool {
 	return false
 }
 
+func (driver *Driver) listPartitionTables(ctx context.Context, databaseName string) (map[db.TableKey][]*storepb.TablePartitionMetadata, error) {
```

Comment:
Also any reason not using the transaction passed from caller?

Reply:
We do not use transaction in sync yet